CHANGELOG.md

```diff
@@ -1,9 +1,120 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
```

Comment:
These should remain in CHANGELOG, I guess... Not in template, but top of this file.

Reply:
Oook... ook.... but the template should contain it because the script first removes the top part and the appends the template on the top :D like it was earlier :D